typescript/sdk/src/token/adapters/CosmWasmTokenAdapter.ts

```diff
@@ -154,6 +161,13 @@ export class CwTokenAdapter
     return false;
   }
 
+  async isRevokeApprovalRequired(
```

Comment:
should this be in `BaseAppAdapter`?

Reply:
I saw that the base adapter was something more generic, to build general adapters for interacting with a specific protocol, that is why I didn't add it there